static/js/click_handlers.js

```diff
@@ -674,6 +675,27 @@ export function initialize() {
 
     // COMPOSE
 
+    $("#compose_giphy_logo").on("click", () => {
+        $("#compose_box_giphy_grid").popover("toggle");
+        let gifs_grid = giphy.renderGIPHYGrid($("#giphy_grid_in_popover .popover-content")[0]);
+        $("body").on(
+            "keyup",
+            "#giphy-search-query",
+            // Use debounce to create a 400ms interval between
+            // every search. This makes the UX of searching pleasant
+            // by allowing user to finish typing before search
+            // begins.
```

Comment:
Maybe "... finish typing before the search is executed."?  I might consider more like 300ms; 400ms is long enough that it might feel laggy.

Reply:
Okay, fair enough.